src/main/java/com/aws/greengrass/builtin/services/pubsub/SubscriptionCallback.java

```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package com.aws.greengrass.builtin.services.pubsub;
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import software.amazon.awssdk.aws.greengrass.model.ReceiveMode;
+
+@Data
+@NoArgsConstructor
+@AllArgsConstructor
+@Builder
+public class SubscriptionCallback {
```

Comment:
I think this should use `@Value` instead of all those others.

Reply:
right. I was thinking if any option would need to become configurable but actually not. Keeping only `@Value` and `@Builder`